<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildGamedataView</class>
 <widget class="QDockWidget" name="BuildGamedataView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Build Gamedata</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="batchExecuteGroupBox">
      <property name="title">
       <string>Batch Execute</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="exportBuildInstallPB">
         <property name="text">
          <string>Export
Build
Install</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="installClientDevPB">
         <property name="text">
          <string>Install
Client Dev</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="installDataShardPB">
         <property name="text">
          <string>Install
Data Shard</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2" rowspan="3">
     <widget class="QGroupBox" name="outputGroupBox">
      <property name="title">
       <string>Output</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="2">
        <widget class="QTextEdit" name="outputTextEdit">
         <property name="documentTitle">
          <string/>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QGroupBox" name="stepByStepGroupBox">
      <property name="title">
       <string>Step-by-Step</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="stepExportPushButton">
         <property name="text">
          <string>1. Export</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stepBuildPushButton">
         <property name="text">
          <string>2. Build</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stepInstallPushButton">
         <property name="text">
          <string>3. Install</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stepDataShardPushButton">
         <property name="text">
          <string>4. Data Shard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stepClientDevPushButton">
         <property name="text">
          <string>5. Client Dev</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stepClientPatchPushButton">
         <property name="text">
          <string>6. Client Patch</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stepClientInstallPushButton">
         <property name="text">
          <string>7. Client Install</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
